### PR TITLE
Add Timeular extension

### PR DESCRIPTION
**Pull request by kudrykv**
## Description

The extension lets you control your time tracking in [Timeular](https://timeular.com/).

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/12697803/149980015-38d5340a-2bed-43cf-b642-17b4894c00a3.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kudrykv 👋 

Thanks for the contribution 💯 

There is a few conflicts:

 - When stopping a timer gives an error toast saying either "Bad Request" or "Conflict" and after stopping it's not possible to start a new

t0jkubgBbqoEYUPJ

 - When starting a timer, it's still possible to open "Start Timer" (which now is empty)

 - Icons is missing
 
k0xfO1XcTgoCtBWQ

Can you start by looking into above? Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by kudrykv** [comment]
Hi @pernielsentikaer!

Thank you for the review!
I will address these.

File: extensions/timeular/src/command-status.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  stop: () => Promise<void>;
+  update: (params: { spaceId: string; activityId: string; text: string; startedAt: string }) => Promise<void>;
+};
+
+const Actions = ({ isLoading, tracking, activity, activities, note, start, update, stop }: ActionsParams) => (
+  <ActionPanel>
+    {tracking && activity && (
+      <PushAction
+        icon={"✏️"}
+        title={"Edit note"}
```
Comment (pernielsentikaer) [inline extensions/timeular/src/command-status.tsx:67]:
```suggestion
        title={"Edit Note"}
```

File: extensions/timeular/src/command-status.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    {tracking && (
+      <ActionPanel.Item
+        icon={"⏹"}
+        title="Stop"
+        onAction={() => stop().catch(showError)}
+        shortcut={{ key: "w", modifiers: ["cmd"] }}
+      />
+    )}
+    {!isLoading && (activity ? activities.filter(a => a.id !== activity.id) : activities).length > 0 && (
+      <ActionPanel.Submenu icon={"▶️"} title="Start Tracking..." shortcut={{ key: "n", modifiers: ["cmd"] }}>
```
Comment (pernielsentikaer) [inline extensions/timeular/src/command-status.tsx:88]:
```suggestion
      <ActionPanel.Submenu icon={"▶️"} title="Start Tracking" shortcut={{ key: "n", modifiers: ["cmd"] }}>
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kudrykv 👋 

That's much better, now I can't reproduce the errors from earlier! Only a few suggestions left, can you address them?


Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @kudrykv 👋 

Thanks for your contribution! Every work as expected 💪 I'm going to accept it and then will my colleague do a final review before it's finally merged.

**Pull request merged**